analysis/bs1116.py

```diff
@@ -44,9 +44,8 @@ def plot_bs1116_results(listeners_dict,
     all_listeners_filename = output_dir + "%s_all_listeners.png" % test_name
     plot_all_listeners(listeners_dict, all_listeners_filename, title=test_name)
 
-    if len(listeners_dict) > 1:
-        all_items_filename = output_dir + "%s_all_items.png" % test_name
-        plot_all(listeners_dict, all_items_filename, title=test_name)
```

Comment:
Now we can always plot "all_listeners" and "all_items" even in the case of having only one result.xml. The script will  create the all_item plot with no error bars essentially.